plutus-ledger-api/src/Plutus/V1/Ledger/Scripts.hs

```diff
@@ -194,8 +194,8 @@ evaluateScript s = do
     case result of
         Right _ -> Haskell.pure ()
         Left errWithCause@(ErrorWithCause err _) -> throwError $ case err of
-            InternalEvaluationError {} -> EvaluationException $ Haskell.show errWithCause
-            UserEvaluationError {}     -> EvaluationError logOut -- TODO fix this error channel fuckery
+            InternalEvaluationError {}    -> EvaluationException $ Haskell.show errWithCause
```

Comment:
would also be good to get it here, although in this case we're doing something a bit odd, maybe it should just be the same as the case below?